TunnelVisionLabs.ReferenceAssemblyAnnotator/Program.cs

```diff
@@ -71,7 +71,7 @@ internal static void Main(TaskLoggingHelper? log, string referenceAssembly, stri
             attributesOfInterest.Add(notNullIfNotNullAttribute.FullName, notNullIfNotNullAttribute);
             attributesOfInterest.Add(notNullWhenAttribute.FullName, notNullWhenAttribute);
 
-            AnnotateAssembly(log, assemblyDefinition, annotatedAssemblyDefinition, attributesOfInterest);
+            AnnotateAssembly(log?.Helper, assemblyDefinition, annotatedAssemblyDefinition, attributesOfInterest);
```

Comment:
❔ Is it easy to pass `log` through so everywhere will be using the new helper?

Reply:
Yes. I wasn't sure if you'd like the larger diff, but it seems more consistent. :+1: